schemas/apps/repositories/v1/repository-events.json

```diff
@@ -0,0 +1,91 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://console.redhat.com/api/schemas/apps/repositories/v1/repository-events.json",
+  "description": "Event data for Repository Events.",
+  "title": "RepositoryEventData",
+  "type": "object",
+  "properties": {
+    "repositories": {
+      "description": "List of repositories affected by the event",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/Repositories"
+      }
+    },
+    "system": {
```

Comment:
This is optional, does it means that some notifications from repositores won't be trigerred with a system?

Reply:
Oh. I think this was blind copying. I'm not sure this is needed at all?